src/routes/index.ts

```diff
@@ -496,6 +494,15 @@ router.route(config.UPDATE_MANAGING_OFFICER_CORPORATE_URL)
   .get(updateManagingOfficerCorporate.get)
   .post(...validator.managingOfficerCorporate, checkValidations, updateManagingOfficerCorporate.post);
 
+router.route(config.CONFIRM_TO_REMOVE_URL + config.ROUTE_PARAM_BENEFICIAL_OWNER_TYPE + config.ID)
+  .all(
+    authentication,
+    companyAuthentication,
+    navigation.hasGivenValidBODetails
```

Comment:
Seems strange to force the user to add valid data for BO before removing but they should have done so I guess.

Reply:
It's definitely required to stop a user navigating to the route with invalid data,
for example if they navigated to confirm-to-remove/beneficial-owner-of-the-world/1234fakeID, they'll be redirected instead of the controller letting them in and rendering an invalid page